acceptable/_service.py

```diff
@@ -1,248 +1,91 @@
 # Copyright 2017 Canonical Ltd.  This software is licensed under the
 # GNU Lesser General Public License version 3 (see the file LICENSE).
 
-"""acceptable - version Flask views using the Accept header."""
-import functools
-import re
-
-from flask import request
-from werkzeug.exceptions import NotAcceptable
+"""acceptable - Programatic API Metadata for Flask apps."""
 
 
 class AcceptableService:
-
     """Main entry point for a service using acceptable to manage API versions.
 
-    This class manages service-wide options such as the vendor identifier. An
-    instance of this class is required to create an API endpoint.
+    This class manages a set of API endpoints for a given service. An instance
+    of this class is required to create an API endpoint.
     """
+    NAME_ALREADY = 'API {} is already registered in service {}'
```

Comment:
This constant is used in exactly one place, and having it as a constant here makes the assertion message below less clear. I'd scrap the constant and just use the string literal below.

Reply:
Right, the reason was code readability, as I think the constant makes it the assertion read better.

Reply:
Fixed